shared/build.gradle.kts

```
@@ -42,20 +47,100 @@ kotlin {
         val commonMain by getting {
             dependencies {
                 implementation(Dependencies.Koin.CORE)
+
+                with(Dependencies.Ktor) {
+                    implementation(CORE)
+                    implementation(SERIALIZATION)
+                    implementation(LOGGING)
+                    implementation(CIO)
+                    implementation(CONTENT_NEGOTIATION)
+                    implementation(JSON)
+                    implementation(AUTH)
+                    implementation(JSON_API)
+                }
+
+                implementation(Dependencies.Log.NAPIER)
             }
         }
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test"))
+                with(Dependencies.Test) {
+                    implementation(COROUTINES)
+                    implementation(KOTEST_ASSERTIONS)
+                    implementation(KOTLINX_RESOURCES)
+                    implementation(MOCKATIVE)
+                    implementation(TURBINE)
+                }
+            }
+        }
+        val androidUnitTest by getting
+        val androidMain by getting {
+            dependencies {
+                implementation(Dependencies.Koin.ANDROID)
+                implementation(Dependencies.Ktor.ANDROID)
+            }
+        }
+        val iosTest by getting
+        val iosX64Main by getting
+        val iosArm64Main by getting
+        val iosSimulatorArm64Main by getting
+        val iosMain by getting {
+            dependsOn(commonMain)
+            iosX64Main.dependsOn(this)
+            iosArm64Main.dependsOn(this)
+            iosSimulatorArm64Main.dependsOn(this)
+            dependencies {
+                implementation(Dependencies.Ktor.IOS)
```

Please keep `test` below of `main` block 🙏 

```suggestion

        val androidMain by getting {
            dependencies {
                implementation(Dependencies.Koin.ANDROID)
                implementation(Dependencies.Ktor.ANDROID)
            }
        }
        val androidUnitTest by getting

        val iosX64Main by getting
        val iosArm64Main by getting
        val iosSimulatorArm64Main by getting
        val iosMain by getting {
            dependsOn(commonMain)
            iosX64Main.dependsOn(this)
            iosArm64Main.dependsOn(this)
            iosSimulatorArm64Main.dependsOn(this)
            dependencies {
                implementation(Dependencies.Ktor.IOS)
            }
        }
        val iosTest by getting {
            dependsOn(commonTest)
            iosX64Test.dependsOn(this)
            iosArm64Test.dependsOn(this)
            iosSimulatorArm64Test.dependsOn(this)
        }
```